tests/features/custom_page/collection.custom_page.feature

```diff
@@ -0,0 +1,44 @@
+@api
+Feature: "Add custom page" visibility options.
+  In order to manage custom pages
+  As a collection member
+  I need to be able to add "Custom Page" content through UI.
+  Background:
+    Given users:
+    | name            | mail                  | roles         | pass |
+    | Rufus Drumknott | member@example.com    |               | test |
```

Comment:
Name changed. roles column removed. It is normal because the method that creates users does not allow the authenticated role to be passed. So if we want a user with only authenticated role assigned to him, we are passing an empty value. But it can work without it so I'm removing the column.